scss/_variables.scss

```diff
@@ -376,6 +377,18 @@ $light-border-subtle:     $light !default; // Boosted mod: instead of `$gray-200
 $dark-border-subtle:      $dark !default; // Boosted mod: instead of `$gray-500`
 // scss-docs-end theme-border-subtle-variables
 
+// Boosted mod
+// Semantic tokens
+
+// Opacity
```

Comment:
Feedback from @louismaximepiton 
We could use directly a map here to better handle adding/removing semantic concepts in B-Brands.

Reply:
I confirm, having the values directly declared in utilities is not handy. I had to create a local copy of the utilities file in order to remove some values.

Reply:
Here's a separate commit that shows how it could be implemented based on this feedback: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2573/commits/b530b18c78155e71bab768550492dff1d22df624
Is this what you had in mind @louismaximepiton and @vprothais?

Note: it can be more difficult to generate from Style Dictionary, but would a problem for our future selves 😄

Reply:
yes, I find it better. If we could use only map in utilities it would be great